src/modules/test/recipes/adhs.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
```

Comment:
Please keep the original recipe file names, do not change. We need to review the delta, for example, also those names were decided upon.